### ISSUE TITLE
Why Yolo OBB downloads YoloN internally?

### ISSUE THREAD

**Issue by shanalikhan** [ultralytics/ultralytics#17516]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I'm trying to finetune `x` OBB model but in the logs everytime it downloads `n` BB model. Here is the code:

```python
model = None
data_version = 'v3_obb'
model_name = 'yolo11x-obb'
device = "0"
batch = 4
lr0 = 0.01
optimizer = 'AdamW'
start_epoch = 0
name = model_name + '_'+ str(batch) +'_' + str(lr0) +'_'+ str(optimizer) + '_' + str(start_epoch)
model = YOLO(model_name+ ".yaml")
print(name).  ---- # yolo11x-obb_4_0.01_AdamW_0

model.load(f"https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11x-obb.pt")

results = model.train(data="/content/v3_yolo8obb_4590/yoloobb_main.yaml",epochs=100, imgsz=512, optimizer=optimizer,name=name,
                      device=device, lr0=lr0,verbose=True, plots=True,save=True,batch=batch,resume=True)
```

Logs are :
In last time it will download `n` model. do we need to setup explicitly to download `x` model of BB for OBB model as well?

```
Ultralytics 8.3.29 🚀 Python-3.10.12 torch-2.5.0+cu121 CUDA:0 (Tesla T4, 15102MiB)
engine/trainer: task=obb, mode=train, model=yolo11x-obb.yaml, data=/content/v3_yolo8obb_4590/yoloobb_main.yaml, epochs=100, time=None, patience=100, batch=4, imgsz=512, save=True, save_period=-1, cache=False, device=0, workers=8, project=None, name=yolo11x-obb_4_0.01_AdamW_03, exist_ok=False, pretrained=https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11x-obb.pt, optimizer=AdamW, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=None, amp=True, fraction=1.0, profile=False, freeze=None, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, show_boxes=True, line_width=None, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=True, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, bgr=0.0, mosaic=1.0, mixup=0.0, copy_paste=0.0, copy_paste_mode=flip, auto_augment=randaugment, erasing=0.4, crop_fraction=1.0, cfg=None, tracker=botsort.yaml, save_dir=runs/obb/yolo11x-obb_4_0.01_AdamW_03

                   from  n    params  module                                       arguments                     
  0                  -1  1      2784  ultralytics.nn.modules.conv.Conv             [3, 96, 3, 2]                 
  1                  -1  1    166272  ultralytics.nn.modules.conv.Conv             [96, 192, 3, 2]               
  2                  -1  2    389760  ultralytics.nn.modules.block.C3k2            [192, 384, 2, True, 0.25]     
  3                  -1  1   1327872  ultralytics.nn.modules.conv.Conv             [384, 384, 3, 2]              
  4                  -1  2   1553664  ultralytics.nn.modules.block.C3k2            [384, 768, 2, True, 0.25]     
  5                  -1  1   5309952  ultralytics.nn.modules.conv.Conv             [768, 768, 3, 2]              
  6                  -1  2   5022720  ultralytics.nn.modules.block.C3k2            [768, 768, 2, True]           
  7                  -1  1   5309952  ultralytics.nn.modules.conv.Conv             [768, 768, 3, 2]              
  8                  -1  2   5022720  ultralytics.nn.modules.block.C3k2            [768, 768, 2, True]           
  9                  -1  1   1476864  ultralytics.nn.modules.block.SPPF            [768, 768, 5]                 
 10                  -1  2   3264768  ultralytics.nn.modules.block.C2PSA           [768, 768, 2]                 
 11                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 12             [-1, 6]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 13                  -1  2   5612544  ultralytics.nn.modules.block.C3k2            [1536, 768, 2, True]          
 14                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 15             [-1, 4]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 16                  -1  2   1700352  ultralytics.nn.modules.block.C3k2            [1536, 384, 2, True]          
 17                  -1  1   1327872  ultralytics.nn.modules.conv.Conv             [384, 384, 3, 2]              
 18            [-1, 13]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 19                  -1  2   5317632  ultralytics.nn.modules.block.C3k2            [1152, 768, 2, True]          
 20                  -1  1   5309952  ultralytics.nn.modules.conv.Conv             [768, 768, 3, 2]              
 21            [-1, 10]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 22                  -1  2   5612544  ultralytics.nn.modules.block.C3k2            [1536, 768, 2, True]          
 23        [16, 19, 22]  1   5055862  ultralytics.nn.modules.head.OBB              [1, 1, [384, 768, 768]]       
YOLO11x-obb summary: 656 layers, 58,784,086 parameters, 58,784,070 gradients, 203.8 GFLOPs

Transferred 1057/1057 items from pretrained weights
TensorBoard: Start with 'tensorboard --logdir runs/obb/yolo11x-obb_4_0.01_AdamW_03', view at http://localhost:6006/
Freezing layer 'model.23.dfl.conv.weight'
AMP: running Automatic Mixed Precision (AMP) checks...
Downloading https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11n.pt to 'yolo11n.pt'...
```

**Comment by UltralyticsAssistant** [comment]
👋 Hello @shanalikhan, thank you for reaching out and for your interest in Ultralytics 🚀! This is an automated response to guide you while an Ultralytics engineer reviews your inquiry.

We recommend you visit our [Docs](https://docs.ultralytics.com) for comprehensive guides and examples which might address your question. You'll find useful information under both the [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) sections.

Since your issue seems related to an unexpected model download behavior, if this is a 🐛 Bug Report, please ensure you provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to assist in diagnosing the issue efficiently.

For any questions related to custom training, please offer additional context, such as dataset samples and training logs. Make sure you follow our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Connect with our growing Ultralytics community! Jump into real-time conversations on [Discord](https://ultralytics.com/discord) 🎧 or engage in broader discussions on our [Discourse](https://community.ultralytics.com) forum. Join us on [Reddit](https://reddit.com/r/ultralytics) to share insights and learn from others.

## Upgrade

To ensure your issue isn't due to outdated software, please upgrade to the latest `ultralytics` package along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a Python environment version 3.8 or higher:

```bash
pip install -U ultralytics
```

## Environments

You can run YOLO in several verified environments:

- Notebooks with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- Google Cloud Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- Amazon Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- Docker Image for more control. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

This badge will show as green if all CI tests are passing. CI tests verify the correctness of YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across different operating systems daily and on every commit.

Your patience is appreciated while we investigate further. An Ultralytics engineer will provide additional assistance soon. 💡
